### ISSUE TITLE
extra space in OS release

### ISSUE THREAD

**Issue by medyagh** [kubernetes/minikube#7167]
remove extra space in none OS release 
```
$ sudo minikube start --vm-driver=none
😄  minikube v1.8.2 on Ubuntu 18.04
✨  Using the none driver based on existing profile
🤹  Running on localhost (CPUs=2, Memory=7470MB, Disk=9749MB) ...
ℹ️   OS release is Ubuntu 18.04.3 LTS
🐳  Preparing Kubernetes v1.17.3 on Docker 19.03.6 ...
    ▪ kubelet.resolv-conf=/run/systemd/resolve/resolv.conf
    > kubectl.sha256: 65 B / 65 B [--------------------------] 100.00% ? p/s 0s
    > kubelet.sha256: 65 B / 65 B [--------------------------] 100.00% ? p/s 0s
    > kubeadm.sha256: 65 B / 65 B [--------------------------] 100.00% ? p/s 0s
    > kubectl: 41.48 MiB / 41.48 MiB [--------------] 100.00% 484.17 MiB p/s 0s
    > kubeadm: 37.52 MiB / 37.52 MiB [--------------] 100.00% 368.95 MiB p/s 0s
    > kubelet: 106.42 MiB / 106.42 MiB [------------] 100.00% 163.16 MiB p/s 1s
🚀  Launching Kubernetes ..
```

**Comment by afbjorklund** [comment]
Where was the extra space ?

**Comment by afbjorklund** [comment]
You mean the one after the icon ? I thought it was added on purpose, in 1f48364487cc95f6921df555cdfba59a3ffcebd5

**Comment by afbjorklund** [comment]
Can someone with a Mac verify this ? i.e. if the text above looks OK, when viewed in terminal

These are the emoji with an extra space after them:
```
	Issues:        {Prefix: "⁉️   "},
	Provisioner:      {Prefix: "ℹ️   "},
	DryRun:           {Prefix: "🏜️   "},
```

We might need to have some platform-specific flag, whether to add extra space or not ?

**Comment by sharifelgamal** [comment]
dry-run looks correct on mac:
<img width="452" alt="Screen Shot 2020-03-24 at 12 45 22 PM" src="https://user-images.githubusercontent.com/20781941/77470326-ba53d180-6dcd-11ea-9765-319340da2c3d.png">

I'm thinking DryRun and Issues are going to need platform specific entries.

Provisioner is only used with the none-driver, so what it looks like on mac isn't relevant, which means this PR is correct.

**Comment by priyawadhwa** [comment]
Closing, as this seems to have been resolved by #7214

**Comment by priyawadhwa** [comment]
Whoops, reopening as #7214 wasn't merged
